src/components/Header/Header.scss

```diff
@@ -154,7 +162,13 @@
     -ms-border-radius: 50%;
     -o-border-radius: 50%;
     box-shadow: 0 0 2px 0px rgba(0, 0, 0, 0.3);
+    /* &:hover {
+      .header-popup {
+        display: block;
+      }
+    } */
```

Comment:
@nkpremices Would you mind removing the above commented code?

Reply:
Noted @gracelungu